library/components/core/WriterTask.bs

```diff
@@ -87,7 +87,7 @@ function getWriteableFile(eventSize as integer) as string
     filenames = ListDir(folderPath).ToArray()
     filenames.Sort("r")
     if (filenames.count() > 0)
-        lastFilename = filenames[0]
+        lastFilename = ""
```

Comment:
If I understand correctly, this will make `fileTimestamp& = strToLong(lastFilename)` to be 0 if there is only ignored file exists (no other batch files), making `uploadTimestamp& > currentTimestamp&` evaluate to `false`?

Reply:
Yes exactly